message_sender/models.py

```diff
@@ -106,3 +106,14 @@ class OutboundSendFailure(models.Model):
 
     def __str__(self):  # __unicode__ on Python 2
         return str(self.id)
+
+
+@python_2_unicode_compatible
+class IdentityLookup(models.Model):
+
+    msisdn = models.CharField(
+        primary_key=True, max_length=255, db_index=True)
+    identity = models.CharField(max_length=36, null=True, blank=True)
```

Comment:
I'm not sure that we want to allow null/blank values here? Otherwise we might update messages so that we lose both the msisdn and the identity.